### PR TITLE
MWPW-142630 - [Marquee] media style CLS

### PR DESCRIPTION
**Pull request by Sartxi**
Pages with both Marquee and Media blocks now have cls scores due to the media.css `.media` padding style being applied to `.marquee .media`. This PR updates the Marquee's `.media` classname to instead be `.asset` to avoid future collisions.
I also shuffled some styles around to fix style lint errors and formatting.

Resolves: [MWPW-142630](https://jira.corp.adobe.com/browse/MWPW-142630)

**MILO**
- Before: https://main--milo--adobecom.hlx.page/drafts/sarchibeque/marquee-cls?martech=off
- After: https://sarchibeque-mwpw-142630-marquee--milo--adobecom.hlx.page/drafts/sarchibeque/marquee-cls?martech=off

**BACOM**
- Before: https://main--bacom--adobecom.hlx.page/products/experience-manager/adobe-experience-manager?martech=off
- After: https://main--bacom--adobecom.hlx.page/products/experience-manager/adobe-experience-manager?milolibs=sarchibeque-MWPW-142630-marquee&martech=off

**BACOM CLS SCORE**
CLSBeforeAfter

### CODE REVIEW

**Comment by rgclayton** [comment]
Adding a CC based test page:
https://main--cc--adobecom.hlx.page/drafts/rclayton/marquee-media?milolibs=sarchibeque-MWPW-142630-marquee

Do we need to communicate the change from `.media` to `.asset` in milo-dev?

**Comment by Sartxi** [comment]
> Adding a CC based test page: https://main--cc--adobecom.hlx.page/drafts/rclayton/marquee-media?milolibs=sarchibeque-MWPW-142630-marquee
> 
> Do we need to communicate the change from `.media` to `.asset` in milo-dev?

Thank you @rgclayton! Yes I will make a post in milo-dev to point out the change. I also added a subtask to the ticket for additional regression testing on this update

**Comment by rgclayton** [comment]
Adding @Blainegunn, @Ruchika4, @JasonHowellSlavin and @meganthecoder for additional eyes. 

If you guys wouldn't mind testing this on your sites to make sure nothing is getting broken.

**Review by meganthecoder — APPROVED** [review]

**Comment by meganthecoder** [comment]
> Adding @Blainegunn, @Ruchika4, @JasonHowellSlavin and @meganthecoder for additional eyes.
> 
> If you guys wouldn't mind testing this on your sites to make sure nothing is getting broken.

@Dli3 Can you verify this on bacom?

**Review by auniverseaway — APPROVED** [review]

**Comment by auniverseaway** [comment]
@Sartxi thanks for looking into this.

**Comment by Ruchika4** [comment]
> Adding @Blainegunn, @Ruchika4, @JasonHowellSlavin and @meganthecoder for additional eyes.
> 
> If you guys wouldn't mind testing this on your sites to make sure nothing is getting broken.

@sigadamvenkata could you please help verify this for CC.

**Review by honstar — APPROVED** [review]

File: libs/blocks/marquee/marquee.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ export default async function init(el) {
     }
 
     let mediaCreditInner;
-    const txtContent = media?.lastChild.textContent.trim();
+    const txtContent = media?.lastChild?.textContent?.trim();
```
Comment (overmyheadandbody) [inline libs/blocks/marquee/marquee.js:117]: This may have already been included in https://github.com/adobecom/milo/pull/1865

**Review by overmyheadandbody — APPROVED** [review]

**Comment by sigadamvenkata** [comment]
we could see one of the marquee block "marquee(split, light)" image alignment changed from left to right on this given test branch libs. 

test page:
before: https://main--cc--adobecom.hlx.page/drafts/venkatas/marquee-media 
after : https://main--cc--adobecom.hlx.page/drafts/venkatas/marquee-media?milolibs=sarchibeque-MWPW-142630-marquee&martech=off

regression urls (you can see the last block in page)
https://main--cc--adobecom.hlx.page/drafts/venkatas/regression-pages/blocks/marquee
https://main--cc--adobecom.hlx.page/drafts/venkatas/regression-pages/blocks/marquee?milolibs=sarchibeque-MWPW-142630-marquee&martech=off

observed the same in milo aswell.

image

cc @Ruchika4  @Sartxi

**Comment by Sartxi** [comment]
> we could see one of the marquee block "marquee(split, light)" image alignment changed from left to right on this given test branch libs.
> 
> test page: before: https://main--cc--adobecom.hlx.page/drafts/venkatas/marquee-media after : https://main--cc--adobecom.hlx.page/drafts/venkatas/marquee-media?milolibs=sarchibeque-MWPW-142630-marquee&martech=off
> 
> regression urls (you can see the last block in page) https://main--cc--adobecom.hlx.page/drafts/venkatas/regression-pages/blocks/marquee https://main--cc--adobecom.hlx.page/drafts/venkatas/regression-pages/blocks/marquee?milolibs=sarchibeque-MWPW-142630-marquee&martech=off
> 
> observed the same in milo aswell.
> 
> image
> 
> cc @Ruchika4 @Sartxi

Great find! Thank you. I pushed a fix for this.

File: libs/blocks/marquee/marquee.js (hunk 1 above)
Comment (Sartxi) [reply libs/blocks/marquee/marquee.js:117]: Thanks! Yes oddly we both found this issue at the same time

**Comment by Ruchika4** [comment]
> we could see one of the marquee block "marquee(split, light)" image alignment changed from left to right on this given test branch libs.
> 
> test page: before: https://main--cc--adobecom.hlx.page/drafts/venkatas/marquee-media after : https://main--cc--adobecom.hlx.page/drafts/venkatas/marquee-media?milolibs=sarchibeque-MWPW-142630-marquee&martech=off
> 
> regression urls (you can see the last block in page) https://main--cc--adobecom.hlx.page/drafts/venkatas/regression-pages/blocks/marquee https://main--cc--adobecom.hlx.page/drafts/venkatas/regression-pages/blocks/marquee?milolibs=sarchibeque-MWPW-142630-marquee&martech=off
> 
> observed the same in milo aswell.
> 
> image
> 
> cc @Ruchika4 @Sartxi

Thanks for noticing that @sigadamvenkata . I see @Sartxi has fixed it now. Please check once from your side also.

**Review by Ruchika4 — APPROVED** [review]

**Comment by sigadamvenkata** [comment]
thanks @Sartxi for quick fix. we could see the alignment is proper now in above mentioned pages.

cc: @Ruchika4

**Review by ryanmparrish — APPROVED** [review]

**Comment by Sartxi** [comment]
@skumar09 I am seeing failures in Nala Tests due to this change in classname from `.media` to `.asset`. Just added you as a reviewer to make sure you are aware of these updates.

**Comment by auniverseaway** [comment]
@nateekar do we have your approval to merge this?

**Comment by Dli3** [comment]
Verified on bacom. Looks good to me. 
cc @Sartxi @meganthecoder

**Comment by skumar09** [comment]
> @skumar09 I am seeing failures in Nala Tests due to this change in classname from `.media` to `.asset`. Just added you as a reviewer to make sure you are aware of these updates.

@Sartxi: Thanks for the heads up ( sorry for the late reply; I was traveling). I have raised the PR for the corresponding Nala script fix.

**Pull request merged**